packages/valory/skills/agent_performance_summary_abci/behaviours.py

```diff
@@ -122,9 +122,9 @@ def calculate_roi(self):
             or trader_agent.get("totalPayout") is None
         ):
             self.context.logger.warning(
-                f"Trader agent data not found or incomplete for {agent_id=} and {trader_agent=}"
+                f"Trader agent data not found or incomplete for {agent_safe_address=} and {trader_agent=}"
             )
-            return None, None
+            return 0, 0
```

Comment:
So, if there is an issue with data, we return a valid response of ROI equal to `0`?